### ISSUE TITLE
HeaderNavMenu Warning

### ISSUE THREAD

**Issue by arcfide** [IBM/carbon-components-svelte#184]
I get the following warning when building with HeaderNavMenu, which I think can be removed:

```
A11y: 'javascript:void(0)' is not a valid href attribute
32:     tabindex="0"
33:     aria-label={text}
34:     href="javascript:void(0)"
        ^
35:     on:keydown
36:     on:keydown={({ key }) => {
```
Surely there is an alternative formulation here?

**Comment by metonym** [comment]
Since v3.22.0, svelte will [emit a warning for `javascript:void(0)`](https://github.com/sveltejs/svelte/pull/4733).

A temporary workaround is to disable the warning:

```
<!-- svelte-ignore a11y-invalid-attribute -->
```

But the long term fix is to use something like `href="#"` and prevent the default behavior

**Comment by arcfide** [comment]
Is there a reason this can't be changed right now? 

https://github.com/IBM/carbon-components-svelte/blob/721968ad2879ddc6efcfe76a289887e4d754298d/src/components/UIShell/GlobalHeader/HeaderNavMenu.svelte#L34

Will it break something?

**Comment by metonym** [comment]
Thanks again for raising the issue.

To resolve the a11y warning, `"javascript:void(0)"` is replaced with an exported prop `href` with a default value of `"/"`. The default click behavior is prevented.

This allows the user to override the underlying `href` value.

Released in v0.6.3 – [changelog notes](https://github.com/IBM/carbon-components-svelte/blob/master/CHANGELOG.md#063---2020-05-26)
